web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -242,6 +242,9 @@ public static void startOpenmrs(ServletContext servletContext) throws ServletExc
 			Listener.loadBundledModules(servletContext);
 
 			Context.startup(getRuntimeProperties());
+			
+			// rebuild lucene search indexes
```

Comment:
@jdegraft this will keep rebuilding the index every time openmrs starts. The ticket is about building it on the initial install.